test/parallel/test-buffer-read.js

```diff
@@ -140,3 +140,28 @@ assert.throws(() => Buffer.allocUnsafe(8).readFloatLE(-1), RangeError);
   assert.strictEqual(buf.readIntLE(0, 6), 0x060504030201);
   assert.strictEqual(buf.readIntBE(0, 6), 0x010203040506);
 }
+
+// https://github.com/nodejs/node/issues/8724 - specific to methods dealing
+// with floating point types because they call out to C++ code.
+//
+// No matching of error messages is done.  We are interested in establishing
+// that the calls don't cause segfaults, the exact error message is irrelevant.
+/* eslint-disable no-restricted-syntax */
+for (const noAssert of [true, false]) {
+  // Out of bounds.
+  assert.throws(() => Buffer.from(1).readFloatBE(0, noAssert));
```

Comment:
If I'm not mistaken, this and the subsequent lines are throwing on `Buffer.from()` (because it can't accept a number as an argument) and never getting to `readFloatBE()`. (Maybe create the Buffer external to the `assert.throws()` to avoid that possibility in future refactoring etc.?)